docs/pages/route-string.page.server.mdx

```diff
@@ -50,13 +50,14 @@ If you define multiple globs, then the glob values are available at `pageContext
 You can use a <Link href="#globs">glob</Link> to catch all URLs:
 
 ```js
-// catch-all.page.route.js
+// /pages/catch-all/+route.js
```

Comment:
@brillout is this the right way of doing `catch-all` in V1?

Reply:
Yes, this also works (as long as the Route String value set by `+route.js` is `/*` or `*`).

Reply:
Ah yes, I got confused for a moment and felt like `catch-all` was a special string, but it can be anything of course. Thanks

Reply:
👍